tests/test_all.py

```diff
@@ -576,13 +581,18 @@ def test_collect(self):
                 }
             }
         }
+        yaml_fetch_properties = {
```

Comment:
since this is common for both the tests in this class, can we make it a class variable ?

Reply:
I would like to keep it separate, because that allows more easily to expand the tests with variations on its content.